src/java.desktop/share/classes/javax/swing/plaf/synth/SynthGraphicsUtils.java

```diff
@@ -553,12 +554,7 @@ static Dimension getPreferredMenuItemSize(SynthContext context,
      }
 
     static void applyInsets(Rectangle rect, Insets insets, boolean leftToRight) {
-        if (insets != null) {
```

Comment:
This method is used only once. Can we remove it and call SwingUtilities3.applyInsets directly instead?

Reply:
Yep! I was about to write the same suggestion.